File: Code/Components/DanpheEMR.ServerModel/ClinicalModels/PatientClinicalInfoModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DanpheEMR.ServerModel.ClinicalModels
{
    public class PatientClinicalInfo
    {
        [Key]
        public int InfoId { get; set; }

        public int PatientId { get; set; }
        public int PatientVisitId { get; set; }
        public string KeyName { get; set; }
        public string Value { get; set; }
        public int CreatedBy { get; set; }
        public DateTime CreatedOn { get; set; }
        public int? ModifiedBy { get; set; }
        public DateTime? ModifiedOn { get; set; }
        public bool IsActive { get; set; }

    }
}

File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/clinical/shared/patient-clinical-info.model.ts
export class PatientClinicalInfoModel {
    public InfoId: number = 0;
    public PatientId: number;
    public PatientVisitId: number;
    public KeyName: string = "";
    public Value: string = "";
    public CreatedBy: number;
    public CreatedOn: string;
    public ModifiedBy: number;
    public ModifiedOn: string;
    public IsActive: boolean;

}
